lib/publiccloud/provider.pm

```diff
@@ -556,12 +554,31 @@ sub get_state_from_instance
     die('get_state_from_instance() isn\'t implemented');
 }
 
+
+=head2 get_container_registry_prefix
```

Comment:
need to remove? It's the same code you added to the new file `k8s_provider.pm`, isn't it?

Reply:
Yes, but now it is required by Google tests. So the best place is in the gcp until we have gcp_client in next PR